init_docs/Text.json

```diff
@@ -383,7 +383,7 @@
    "Not_Set": "Not Set",
    "Course_Information": "Course Information",
    "level_Single":"Level",
-   "Hi":"Hi",
+   "Hi":"Hello",
```

Comment:
hmm - this gets changed but then never used the whole PR-code

please change it back

cause it looks like in line 840 you created `Hello`